### DISCUSSION TITLE
Étude de cas : "Histoire du cinéma"

### DISCUSSION THREAD

**Discussion by Antoine-ValentinCharpentier** [Hypertopic/HyperGlosae#134] [Show and tell]
Here is the video presenting all the content we added to Hyperglosae during this semester.

The video was too heavy for GitHub, so we uploaded it to YouTube, click on the screenshot to open it.

<a href="https://youtu.be/hQx4ex8X21E"><img width="1119" alt="Capture d’écran 2024-01-24 à 16 50 40" src="https://github.com/Hypertopic/HyperGlosae/assets/597307/941a86c8-5f30-478a-89c1-66d92b9a1501"></a>

**Comment by benel** [comment]
Merci à toute l'équipe pour votre contribution à la collaboration avec M. Laborderie et au logiciel lui-même.

Je laisse la parole à M. Laborderie pour des commentaires ou des questions.

**Reply by P-Laborderie** [reply]
Bonjour à toutes et à toutes,

 et tout d'abord mes félicitations pour l'élaboration de ce logiciel !

Pour mémoire, en tant que chercheur en cinéma et audiovisuel, précisément en histoire du cinéma, j'ai discuté avec vous à deux reprises pour identifier des  besoins concernant les fonctionnalités d'un logiciel qui permette de favoriser la recherche sur les films, notamment les suivantes:
 - marquer des points repérés sur un film,
 - les annoter avec des mots-clés et des descriptions courtes
 - pouvoir ensuite naviguer entre ces différents points.
 
Un autre enjeu consisterait à construire un thesaurus à partir de ces mots-clés.
Une autre fonctionnalité consisterait à faire une recherche par mots-clés dans un corpus de plusieurs films.

Outre l'archivage de films à destination des historien du cinéma et de l'audiovisuel, ces fonctionnalités seraient exploitables dans deux autres contextes:
- certains chercheurs en sciences humaines et sociales filment leurs expérimentations (par exemple, en sciences de l'éducation, le filmage du déroulement d'une séance pédagogique en classe). Il est ensuite intéressant de pouvoir facilement utiliser sur ce type de document audiovisuel les diverses fonctions que vous avez programmées.
- enfin, il serait intéressant de développer votre logiciel en ciblant un public d'enfants dans un contexte d'éducation à l'image. Dans cette perspective, il faudrait rendre les boutons d'accès aux fonctions plus ludiques (avec des pictogrammes, par exemple). Autre idée : pouvoir effectuer une recherche qui regroupe toutes les interventions qu'un utilisateur aurait faites et qui pourraient apparaître dans son profil.

Encore bravo !

**Comment by philacour** [comment]
Très intéressant, merci. 
Je me demande en particulier comment utiliser ces développements pour promouvoir le sous-titrage multilingue collaboratif, notamment avec TraduXio. L'insertion de commentaires en marges des vidéos est un excellent appui éducatif. Stielger et son équipe de prospective du Centre Pompidou avait naguère conçu un système un peu similaire (lignes de temps), mais beaucoup moins intuitif et convivial. Félicitations, donc : c'est très prometteur
